hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/viewfs/InodeTree.java

```diff
@@ -61,7 +61,7 @@
 
 @InterfaceAudience.Private
```

Comment:
Public?

Reply:
Owen recommended to keep the interface audience as private the other day.